stand/defaults/loader.conf.5

```diff
@@ -140,6 +140,27 @@ present file.
 should be treated as write-only.
 One cannot depend on any value remaining in the loader environment or carried
 over into the kernel environment.
+.It Ar product_vars
+When set, must be a space separated list of environment variable names to walk
+through to guess the product.
+
+And if a product can be guessed, prepend:
+.Bl -bullet -compact
+.It
+.Pa /boot/PRODUCT.conf
+to
+.Ar loader_conf_files
+.It
+and
+.Pa /boot/loader.conf.d/PRODUCT
+to
+.Ar loader_conf_dirs
+.El
+
+It can be typically used as follow:
+.Pp
+.Dl product_vars="smbios.system.product"
+
```

Comment:
If you take up my suggestion below, then you'd need to tweak this to note it is done for all matches.

Reply:
oh, and bump .Dd at the top please.